src/dashboard/helpers/general.js

```diff
@@ -29,3 +29,29 @@ export const CONTRACTS = {
     kovan: '0xbF7A7169562078c96f0eC1A8aFD6aE50f12e5A99'
   }
 };
+
+export const DISCOVERABLE_PROVIDERS = {
+  oneClickDapp: {
+    name: 'One Click Dapp',
+    address: '0xb4377efc05bd28be8e6510629538e54eba2d74e3',
+    website: 'http://oneclickdapp.com',
+    description: 'Generate frontends quickly for your smart contracts',
+    tags: ['open-source', 'tools', 'web3'],
+    image: 'oneClickDapp'
+  },
+  superMarianoBrothers: {
+    name: 'Super Mariano Bros.',
+    description:
+      'Pick up coins as you make your way through Governance castle.',
+    address: '0xbF7A7169562078c96f0eC1A8aFD6aE50f12e5A99',
+    website: './',
+    tags: ['game', 'paid', 'entertainment'],
+    image: 'superMariano'
+  }
+};
+
+export const NETWORKS = {
+  kovan: {
+    etherscan: 'https://kovan.etherscan.io/address/'
+  }
+};
```

Comment:
This file feels like it's misnamed
`helper/general.js` I think it's better to denote these as constants no?

Reply:
done